src/runtime/components/NuxtTime.vue

```diff
@@ -38,30 +38,7 @@ const date = computed(() => {
   return new Date(props.datetime)
 })
 
-const useLocale = computed(() => locale ?? props.locale)
-const formatter = computed(() => {
-  return new Intl.DateTimeFormat(useLocale.value, {
-    localeMatcher: props.localeMatcher,
-    weekday: props.weekday,
-    era: props.era,
-    year: props.year,
-    month: props.month,
-    day: props.day,
-    hour: props.hour,
-    minute: props.minute,
-    second: props.second,
-    timeZoneName: props.timeZoneName,
-    formatMatcher: props.formatMatcher,
-    hour12: props.hour12,
-    timeZone: props.timeZone,
-    calendar: props.calendar,
-    dayPeriod: props.dayPeriod,
-    numberingSystem: props.numberingSystem,
-    dateStyle: props.dateStyle,
-    timeStyle: props.timeStyle,
-    hourCycle: props.hourCycle,
-  })
-})
+const formatter = computed(() => new Intl.DateTimeFormat(locale ?? props.locale, props))
```

Comment:
changing the date value will fire formatter creation?.

Reply:
confirmed, changing date also recreates the formatter twice

Reply:
changing locale also not reactive with your proposal

Reply:
I don't believe this was reactive, even before my refactor, but I might be wrong. We should do that, but we can do it in another PR.

Reply:
because you're always using locale from html, you should split it with process.server and invert the usage:
```ts
const formatter = process.server
  ? computed(() => locale)
  : computed(() => new Intl.DateTimeFormat(props.locale ?? locale, props))
```

Reply:
with this change, it works, but current playground will change the locale after the timeout (on F5, on lcient just works), I don't know what is the purpose of that timeout...

Reply:
Yes, there was a hydration mismatch caused by that. It's important not to switch the source of the locale/date until the component is mounted.